relecov_tools/sftp_client.py

```
@@ -165,12 +165,16 @@ def get_file_list(self, folder_name):
         """
         log.info("Listing files in %s", folder_name)
         file_list = []
-        content_list = self.sftp.listdir_attr(folder_name)
-        file_list = [
-            os.path.join(folder_name, content.filename)
-            for content in content_list
-            if stat.S_ISREG(content.st_mode)
-        ]
+        try:
+            content_list = self.sftp.listdir_attr(folder_name)
+            for content in content_list:
+                full_path = os.path.join(folder_name, content.filename)
+                if stat.S_ISDIR(content.st_mode):
+                    file_list.extend(self.get_file_list(full_path))
+                elif stat.S_ISREG(content.st_mode):
+                    file_list.append(full_path)
+        except FileNotFoundError:
+            log.error(f"Folder not found: {folder_name}")
```

Do not hide the error here, include `raise` after log.error